lib/web/url.py

```diff
@@ -211,12 +211,12 @@ def open(url, wait=10):
                 o = urllib2.build_opener(p, urllib2.HTTPHandler)
                 urllib2.install_opener(o)
             connection = urllib2.urlopen(request)
```

Comment:
I think all these exception could be along the lines of `except urlib.XYZ as e`  ?